feature-libs/product/configurators/rulebased/components/show-more/configurator-show-more.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="cx-show-more" *ngIf="text">
+  <div #textContent class="cx-show-more-text">
+    <span> {{ textToShow }}</span>
```

Comment:
is the `space` at the beginning necessary?

Reply:
removed